.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('gts/.prettierrc.json')
```

Comment:
Is this required?

Reply:
I went ahead and removed it, and updated the local `.prettierrc` with the gts rules.  I also added `prettier` as a dev dependency